### PR TITLE
[BD-18] Standardize the make test targets

### PR DESCRIPTION
**Pull request by emZubair**
## Standardize the make test targets
Added target for test & clean

**Packages added**

- pytest-cov

### CODE REVIEW

**Review by tasawernawaz — APPROVED** [review]
LGTM,
please add new lines at the end files.

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,14 @@
...
+
+# JetBrains
+.idea
+
+# QA
+coverage.xml
+
+# Unit test / coverage reports
+.coverage
+.tox
```
Comment (tasawernawaz) [inline .gitignore]: new line required, on all such instances

Comment (emZubair) [reply .gitignore]: added

File: Makefile [hunk 2]
```diff
@@ -1,6 +1,29 @@
...
+	rm -rf pii_report
+
 upgrade: export CUSTOM_COMPILE_COMMAND=make upgrade
 upgrade: ## update the requirements/*.txt files with the latest packages satisfying requirements/*.in
 	pip install -q -r requirements/pip_tools.txt
 	pip-compile --upgrade -o requirements/pip_tools.txt requirements/pip_tools.in
 	pip-compile --upgrade -o requirements/base.txt requirements/base.in
 	pip-compile --upgrade -o requirements/travis.txt requirements/travis.in
+	pip-compile --upgrade -o requirements/test.txt requirements/test.in
+
```
Comment (ziafazal) [inline Makefile:21]: line space

**Review by ziafazal — APPROVED** [review]
👍

**Review by ziafazal — APPROVED** [review]
👍

**Pull request merged**